mail-filter/mimedefang/files/mimedefang.init

```diff
@@ -0,0 +1,62 @@
+#!/sbin/openrc-run
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+PROGNAME=${SVCNAME#*.}
+PROGDIR=/usr/bin
+SPOOLDIR=${SPOOLDIR:-/var/spool/MIMEDefang}
+PID="${SPOOLDIR}/${PROGNAME}.pid"
+MXPID="${SPOOLDIR}/${PROGNAME}-multiplexor.pid"
+LOCK="${SPOOLDIR}/${PROGNAME}.lock"
+MXLOCK="${SPOOLDIR}/${PROGNAME}-multiplexor.lock"
+
+# Make sure required vars are set
+SOCKET=${SOCKET:-${SPOOLDIR}/${PROGNAME}.sock}
+MX_SOCKET=${MX_SOCKET:-${SPOOLDIR}/${PROGNAME}-multiplexor.sock}
+
+description="MIMEDefang mail filter"
+extra_commands="checkconfig"
+description_checkconfig="Check config for ${SVCNAME}"
+
+depend() {
+	use antivirus
+}
+
+checkconfig() {
+	checkpath -d "${SPOOLDIR}"
+	einfo "multiplexor socket: ${MX_SOCKET}"
+	einfo "sendmail socket: ${SOCKET}"
+}
+
+start() {
+	checkconfig || return 1
+
+	# First start mimedefang multiplexor
+	rm -f ${MX_SOCKET} > /dev/null 2>&1
```

Comment:
Please `""`-quote paths and other vars where you can know there will be no whitespace parameter splitting intended.